README.md

```diff
@@ -85,21 +87,22 @@ python3 -m multicast --help ;
 
 If all went well `multicast` is now installed and working :tada:
 
+
 ### How do I use this to receive some UDP Multicast?
 
 (assuming project is setup and installed and you want to listen on 0.0.0.0)
 
 ```bash
 # cd /MY-AWSOME-DEV-PATH
-python3 -m multicast HEAR --use-std --port 5353 --join-mcast-groups 224.0.0.1 --bind-group 224.0.0.1
+python3 -m multicast HEAR --use-std --port 59595 --join-mcast-groups 224.0.0.1 --bind-group 224.0.0.1
```

Comment:
* The example is better because it uses a dynamic port and thus is more realistic

_Also with lines R105, R116 and 130_